NEWS

```diff
@@ -1,9 +1,13 @@
-iSubGen 1.0.2 2023-11-06
+iSubGen 1.0.2 2024-08-20
+
+UPDATE
+* Allow functions to accept data.frame and a list of data.frames
+* Fixed issues caused by R version updates
 
 ADDED
 * CI/CD actions to test code style and run standard R package checks
 * Standard metadata files for development
-
+* Added standard LICENSE file
```

Comment:
**(question)**
@maotian06 I'm not sure if we've standardized the format of the `NEWS` file in our R package repos but I think we can generally follow our established standards if no objections. https://keepachangelog.com/en/1.1.0/
Has Infra WG recently discussed this? @yashpatel6 @dan-knight ?

Reply:
We haven't; I'm not sure if the format as is was determined based on any standard specific to R packages? @tyamaguchi-ucla @dan-knight

Reply:
This was a little confusing to me. Let's wait until the infra team decides.
I updated the `UPDATE` to `CHANGED` as in the changelog.